docs/resources.html

```diff
@@ -67,6 +68,13 @@ <h4 id="resources-for-editors">Resources for editors</h4>
+				<h3 id="open-peer-review">Open Peer Review</h3>
+				<p>“Open review” means different things depending on what is open and to whom. Open review is a good example of a situation where the most open version may not be the fairest or best option, as there are many factors to take into account. It is also important to distinguish between open identities and open reviews (both of which, confusingly, can be called open reviewing). For a review of the benefits and limitations of open peer review (some outlined below), see <a href="https://doi.org/10.1186/s41073-020-00094-z" target="_blank">Besançon et al. (2020)</a>. For guidelines on implementing open peer review, see <a href="https://doi.org/10.1186/s41073-019-0063-9" target="_blank">Ross-Hellauer and Görögh (2019)</a>.</p>
+				<h4 id="open-identities">Open Identities</h4>
+				<p>Journals will have a policy on whether editors, reviewers, and authors know each other’s identities. In a <b>single-blind</b> system the editor knows the identities of reviewers and authors, reviewers and authors both know the identity of the editor, but authors and reviewers do not know the identity of each other. This is in an attempt to eliminate personal biases of reviewers and authors (e.g. those based on gender, seniority, reputation and affiliation). In a <b>double-blind</b> system even the editor is blind to the identity of the authors, taking into account the possible personal biases of the editor themself. See <a href="https://www.nature.com/" target="_blank">Nature’s editorial</a> adopting an optionally double-blind system and <a href="https://doi.org/10.1073/pnas.1707323114" target="_blank">Tomkins et al (2017)</a> for evidence of single-blind review being biased towards papers with famous authors and authors from high-prestige institutions. In <b>open reviewing</b> everyone knows the identity of everyone. This is in an attempt to encourage more civil and thoughtful reviewer comments. There is no one-size-fits-all solution, but it is important to think carefully about which policy makes sense for your journal.</p>
+				<h4 id="open-reviews">Open Reviews</h4>
+				<p>It is also possible to make the reviews themselves openly available alongside published manuscripts (with or without the reviewers being identified). This can help to give context to the published article.</p>	
+				<a href="https://" target="_blank">XXX</a>
```

Comment:
Looks like something missing here?

Reply:
Ah sorry! Was just using that to copy and paste into where I needed it, so you can delete it. Thanks!!

Reply:
(the last line that is)

Reply:
will do